### ISSUE TITLE
bin/rails zeitwerk:check displays deprecation warning on Rails 6.0.3.4

### ISSUE THREAD

**Issue by mrbongiolo** [hotwired/turbo-rails#15]
```
DEPRECATION WARNING: Initialization autoloaded the constants Turbo::Streams, Turbo::Streams::TurboStreamsTagBuilder, Turbo::Frames, Turbo::Frames::FrameRequest, Turbo::Native, Turbo::Native::Navigation, Turbo::DriveHelper, Turbo::FramesHelper, Turbo::IncludesHelper, Turbo::StreamsHelper, Turbo::Streams::ActionHelper, and Turbo::Broadcastable.

Being able to do this is deprecated. Autoloading during initialization is going
to be an error condition in future versions of Rails.

Reloading does not reboot the application, and therefore code executed during
initialization does not run again. So, if you reload Turbo::Streams, for example,
the expected changes won't be reflected in that stale Module object.

These autoloaded constants have been unloaded.

Please, check the "Autoloading and Reloading Constants" guide for solutions.
 (called from <top (required)> at /config/environment.rb:7)
Hold on, I am eager loading the application.
rails aborted!
```

**Comment by sedubois** [comment]
In my Rails 6.1.0 app I get this:

```
Exiting
Uncaught exception: DEPRECATION WARNING: Initialization autoloaded the constant Turbo::Broadcastable.

Being able to do this is deprecated. Autoloading during initialization is going
to be an error condition in future versions of Rails.

Reloading does not reboot the application, and therefore code executed during
initialization does not run again. So, if you reload Turbo::Broadcastable, for example,
the expected changes won't be reflected in that stale Module object.

This autoloaded constant has been unloaded.

In order to autoload safely at boot time, please wrap your code in a reloader
callback this way:

    Rails.application.reloader.to_prepare do
      # Autoload classes and modules needed at boot time here.
    end

That block runs when the application boots, and every time there is a reload.
For historical reasons, it may run twice, so it has to be idempotent.

Check the "Autoloading and Reloading Constants" guide to learn more about how
Rails autoloads and reloads.
 (called from <main> at .../config/environment.rb:5)
 ...
```

**Comment by fxn** [comment]
I am preparing a patch to address these ones.

**Comment by dhh** [comment]
Fixed via #51.
